internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -103,8 +110,31 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 	return content.String(), nil
 }
 
+func (s *LoadBalancedWebService) loadBalancerTarget() (targetContainer *string, targetPort *string, err error) {
+	containerName := s.name
+	containerPort := strconv.FormatUint(uint64(aws.Uint16Value(s.manifest.Image.Port)), 10)
+	// Route load balancer traffic to main container by default.
+	targetContainer = aws.String(containerName)
+	targetPort = aws.String(containerPort)
+	mftTargetContainer := s.manifest.TargetContainer
+	if mftTargetContainer != nil {
+		sidecar, ok := s.manifest.Sidecars[*mftTargetContainer]
+		if ok {
+			targetContainer = mftTargetContainer
+			targetPort = sidecar.Port
+		} else {
+			return nil, nil, fmt.Errorf("target container %s doesn't exist", *s.manifest.TargetContainer)
+		}
+	}
+	return targetContainer, targetPort, nil
```

Comment:
with named returned values, we can just write `return` instead of  `return targetContainer, targetPort, nil`

Reply:
nice!